code/obj/item/implant.dm

```diff
@@ -533,7 +535,7 @@ THROWING DARTS
 
 					ticker.mode:remove_revolutionary(H.mind)
 				else
-					if (prob(30))
+					if (prob(30) && H.hasStatus("derevving"))
```

Comment:
Not really sure what this check is for

Reply:
Was a part of code from a previous pr, gonna remove